doc/source/user_guide/reshaping.rst

```diff
@@ -726,6 +726,92 @@ To choose another dtype, use the ``dtype`` argument:
 
     pd.get_dummies(df, dtype=bool).dtypes
 
+.. versionadded:: 1.4.0
+
+To convert a "dummy" or "indicator" ``DataFrame``, into a categorical ``DataFrame``,
+for example ``k`` columns of a ``DataFrame`` containing 1s and 0s can derive a
+``DataFrame`` which has ``k`` distinct values
+:func:`~pandas.from_dummies`:
+
+.. ipython:: python
+
+   df = pd.DataFrame({"prefix_a": [0, 1, 0], "prefix_b": [1, 0, 1]})
```

Comment:
Should this df have a missing column, e.g., b is [1, 0, 0], to show the default behavior?

Reply:
Yes it should show the default behavior, I thought that two rows are to little. Maybe expanding it to four or five might fix this potential missunderstanding as it makes it more obvious that there is not just something missing?